canonical/ubuntu.com
# /server/overview align pictos and remove border

## Done

/server/overview centre align IBM logo and pictogram for "fast track to virtualisation” and remove border from last row.
I’ve also removed the partial as this is the only place it’s used now
## QA

Go to /server and make sure the two logos are centered on small and that the border on the last row is no more on medium and large
## Issue / Card

Issue: Fixes #355 
Card: https://trello.com/c/Wykq1Air
